source/openapi-admin-v3.yaml

```diff
@@ -882,13 +884,70 @@ paths:
     parameters:
       - "$ref": "#/components/parameters/GroupId"
       - "$ref": "#/components/parameters/AppId"
+  
+  "/groups/{groupId}/apps/{appId}/services/config":
+    get:
+      tags:
+        - services
+      operationId: adminGetServiceConfig
+      summary: Get configuration of sources and third-party services
+      description: >-
+        Get configuration of [data
+        sources](https://www.mongodb.com/docs/atlas/app-services/mongodb/) 
+        and [third-party services [Deprecated]](https://www.mongodb.com/docs/atlas/app-services/reference/services/).
+      responses:
+        "200":
+          description: Successfully returned service configuration.
+          content:
+            application/json:
+              schema:
+                items:
+                  anyOf:
+                    - $ref: "#/components/schemas/AtlasCluster"
+                    - $ref: "#/components/schemas/AtlasFederatedInstance"
+                    - $ref: "#/components/schemas/ThirdPartyService"
+    patch:
+      tags:
+        - services
+      operationId: adminUpdateServiceConfig
+      summary: Update configuration settings of sources and third-party services
+      description: >-
+        Update the [data
+        source](https://www.mongodb.com/docs/atlas/app-services/mongodb/) or
+        [third-party
+        service [Deprecated]](https://www.mongodb.com/docs/atlas/app-services/reference/services/) 
+        configuration.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: "#/components/schemas/CreateDataSource"
+                - $ref: "#/components/schemas/AtlasFederatedInstance"
+                - $ref: "#/components/schemas/ThirdPartyService"
+      responses:
+        "201":
+          description: Successfully created.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: "#/components/schemas/ServiceResponse"
+                  - $ref: "#/components/schemas/AtlasFederatedInstance"
+                  - $ref: "#/components/schemas/ThirdPartyService"
+        "409":
+          description: There is already a service with the given `name`.
```

Comment:
```suggestion
        "204":
          description: Successfully updated.
```

Looks like this still has the createService responses. Just checked and the update config endpoint returns 204 No Content

<img width="542" alt="Screenshot 2023-10-13 at 5 40 47 PM" src="https://github.com/mongodb/docs-app-services/assets/15657698/1832e4ae-1c5d-42ad-81aa-745474b96d50">